docs/development/releasing.rst

```diff
@@ -12,231 +12,249 @@ more of the process, if not all.
-     upload)" enter 'N': zest.releaser does not offer enough control yet over
-     how the register and upload are performed so we will do this manually
-     until the release scripts have been improved.
+      $ git commit -m "Back to development: v<next_version>"
 
- 13. You will be asked to enter a new development version.  Normally the next
-     logical version will be selected--press enter to accept the default, or
-     enter a specific version string.  Do not add ".dev" to the version, as
-     this will be appended automatically (ignore the message that says ".dev0
-     will be appended"--it will actually be ".dev" without the 0).  For
-     example, if the just-released version was "0.1" the default next version
-     will be "0.2".  If we want the next version to be, say "0.1.1", or "1.0",
-     then that must be entered manually.
+#. Now go back and check out the tag of the released version with
+   ``git checkout v<version>``.  For example::
 
- 14. You will be shown a diff of CHANGES.rst showing that a new section has
-     been added for the new development version, and showing that the version
-     has been updated in setup.py.  Enter 'Y' to commit these changes.
+      $ git checkout v0.2.2
 
- 15. When asked to push the changes to a remote repository, enter 'Y'.  This
-     should complete the portion of the process that's automated at this point.
+   Don't forget to remove any non-commited files with::
 
- 16. Check out the tag of the released version.  For example::
+      $  git clean -dfx
 
-         $ git checkout v0.1
+#. Create the source distribution by doing::
 
- 17. Create the source distribution by doing::
-
-         $ git clean -dfx  # just in case
          $ python setup.py build sdist
 
-     Copy the produced ``.tar.gz`` somewhere and verify that you can unpack it,
-     build it, and get all the tests to pass.  It would be best to create a new
-     virtualenv in which to do this.
-
      .. note::
 
-         In the future, the ``build`` command should run automatically as a
+         In the future, the ``build`` command may run automatically as a
          prerequisite for ``sdist``.  But for now, make sure to run it
          whenever running ``sdist`` to ensure that all Cython sources and
          other generated files are built.
 
- 18. Register the release on PyPI with::
+#. Run the tests in an environment that mocks up a "typical user" scenario.
+   This is not strictly necessary because you ran the tests above, but
+   it can sometimes be useful to catch subtle bugs that might come from you
+   using a customized developer environment.  For more on setting up virtual
+   environments, see :ref:`virtual_envs`, but for the sake of example we will
+   assume you're using `Anaconda`_. Do::
 
-         $ python setup.py register
+      $ conda create -n astropy_release_test_v<verson> numpy
+      $ source activate astropy_release_test_v<verson>
+      $ cd dist
+      $ mkdir test
```

Comment:
good point, and actually this can all be done from straight in the source dir to simplify things